graylog2-web-interface/src/views/pages/ShowViewPage.jsx

```diff
@@ -63,6 +72,7 @@ class ShowViewPage extends React.Component<Props, State> {
 
   loadView = (viewId: string): Promise<?View> => {
     const { location, loadingViewHooks, executingViewHooks, viewLoader } = this.props;
+    // eslint-disable-next-line react/prop-types
```

Comment:
Instead of disabling eslint, could this destruct be part of line 74 destruct?

Reply:
I will look at that!